README.md

```diff
@@ -1,6 +1,10 @@
 
 # The Node.js for Mobile Apps React Native plugin
 
+## Reporting Issues
+
+We have a [central repo](https://github.com/janeasystems/nodejs-mobile/issues) where we manage all the issues related to NodeJS for Mobile Apps, including specific issues for this plugin. Please, report your issue [there](https://github.com/janeasystems/nodejs-mobile/issues).
```

Comment:
`NodeJS` => `Node.js` :smile: 
`specific issues for this plugin` => `issues specific to this plugin`

Reply:
I've addressed the issue in a new commit. PTAL